asv.conf.json

```
@@ -15,7 +15,7 @@
 
     // Install using default qt install
     "build_command": ["python -V"],  // skip build stage
-    "install_command": ["in-dir={env_dir} python -m pip install {conf_dir}[all]"],
+    "install_command": ["in-dir={env_dir} python -m pip install {conf_dir}[all,testing]"],
```

```suggestion
    "install_command": ["in-dir={env_dir} python -m pip install {build_dir}[all,testing]"],
```

Seeing this line in isolation brought my attention to `conf_dir` after a debugging session @kcpevey and I had last week.

It's a subtle but important issue that renders `asv continuous` useless in a way because by using `conf_dir` we are installing napari from the repo _always in the same state_. So no matter which two commits are chosen for `continuous`, they are ignored, and the current HEAD is used for both.

Why? Turns out that `asv` doesn't do `git checkout` in-place, but it first clones the repo elsewhere: `build_dir` ([docs](https://asv.readthedocs.io/en/stable/asv.conf.json.html#build-command-install-command-uninstall-command)). That's where the `checkout` happens.

This means that the benchmark CI might have been giving false positives all along, because it always compared the PR to itself. If anybody was worried about performance regressions in a PR but the CI said otherwise, it should be re-checked.

---

Note `conf_dir` is left as is so the benchmark code itself does not change across runs.

In a way, the commands are like:

```
git clone {conf_dir} {build_dir}
cd {build_dir}
git checkout {commit-A}
asv run {conf_dir}/benchmarks .
git checkout {commit-B}
asv run {conf_dir}/benchmarks .
```